Atea Project/AteaScraperV2/AteaScraper/AteaScraper/Services/AzureStorageService.cs

```diff
@@ -11,37 +10,21 @@ namespace AteaScraper.Services
 {
     public class AzureStorageService : IAzureStorageService
     {
-        private readonly TableServiceClient _tableServiceClient;
-        private readonly BlobContainerClient _blobContainerClient;
-
-        public AzureStorageService(IConfiguration configuration,TableServiceClient tableServiceClient, BlobContainerClient blobContainerClient)
-        {
-            _tableServiceClient = tableServiceClient;
-            _blobContainerClient = blobContainerClient;
-            // Get connection string and container name from configuration
-            string connectionString = "UseDevelopmentStorage = true";
-            string containerName = "atea";
-
-            // Initialize TableServiceClient
-            _tableServiceClient = new TableServiceClient(connectionString);
-
-            // Initialize BlobContainerClient
-            _blobContainerClient = new BlobContainerClient(connectionString, containerName);
-        }
         public async Task<bool> AddEntityToTableAsync(string partitionKey, string rowKey, bool isSuccess, Guid requestKey)
         {
             try
             {
-                var tableClient = _tableServiceClient.GetTableClient("atea");
-                await tableClient.CreateIfNotExistsAsync();
+                var serviceClient = new TableServiceClient("UseDevelopmentStorage=true");
```

Comment:
connection string uz local.settings.json laikam tā bija. 
uz konfigurācijas failu jānes.
arī tabulu nosaukumi.

Reply:
ok paldies, man tik jautajums, vai es atstaju tos servisus azur function, vai taisu libary un tur visu lieku, jo man tad sanak ka vajag visas pacinas ielikt jaunaja class libary

Reply:
labāk class library visu kas pārizmantojams. :) 
kas ir atkal specific.. to var atstāt... bet man liekas ka intergrācijas visas ir pārizmantojamas. :)